components/icons/HomeIcon.tsx

```diff
@@ -0,0 +1,21 @@
+export const HomeIcon = () => {
```

Comment:
I would honestly have all of these icon definitions on the same file and have them all have the same props